api/build.gradle

```diff
@@ -20,11 +20,23 @@
  */
 
 jar {
-  baseName = "ironjacamar-spec"
+  manifest {
+    attributes("Implementation-Title" : "The Java EE Connector Architecture 1.7 API")
+  }
+  baseName = "ironjacamar-spec-api"
   dependsOn classes
   from sourceSets.main.output
 }
 
+//copy dependencies .jar to be collected by the distribution task
```

Comment:
This would be better in the top-level build.gradle file.

It could be a precise list of dependencies too. And should be without their version number in the name, e.g. jandex.jar instead of jandex-2.0.0.Beta1.jar